src/components/DataController/DataController.js

```diff
@@ -172,6 +176,14 @@ var DataController = React.createClass({
               <Col lg={4} lgPush={8} md={6} mdPush={6} sm={6} smPush={6}>
                 <Selector label={"Dataset"} onChange={this.updateAnnCycleDataset} items={ids} />
               </Col>
+              <Col lg={4} lgPush={1} md={6} mdPush={1} sm={6} smPush={1}>
+                <div>
+                  Download Data
```

Comment:
Maybe this should be a [`ControlLabel`](https://react-bootstrap.github.io/components.html#forms-props-control-label) to give it the same styling as the other label elements on the page?